docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
+
+The Docker daemon can be configured to use Linux user namespace support to
+remap the root user inside the container to an unprivileged user on the host.
+Due to the requirement that the filesystem layers need to have user/group
+ownership modification to make this remapping useful, this is a daemon-level
+option and not selectable on a per-container basis.
+
+To enable user namespace support provide a username or Linux uid:gid to the
+`-root` flag.  If you would like the daemon to create and use a default user
+managed by the Docker installation you can specify "default" and a user and
+group named `dockroot` will be created (if it does not already exist) and
+used as the root remapped ID inside all containers for that daemon instance.
+
+Example relying on default Docker username management:
+
+    $ sudo docker -d -r default
```

Comment:
An alternative could be to have `dockroot` be the *default* user, and `dockroot` would be special in the sense that if it's a non-existing user it would be created.